mp2-v1/src/receipt_extraction/mod.rs

```diff
@@ -0,0 +1,31 @@
+pub mod leaf;
+pub mod public_inputs;
+
+use mp2_common::{
+    digest::Digest, eth::EventLogInfo, group_hashing::map_to_curve_point, types::GFp,
+};
+use plonky2::field::types::Field;
+
+/// Calculate `metadata_digest = D(key_id || value_id || slot)` for receipt leaf.
```

Comment:
The code below doesn't seem to reflect this comment. Can you clarify ?

Reply:
Apologies, I was using some of the already existing code as a starting point and adapting it, I must have forgotten to update the comment I'll fix that now